README.md

```diff
@@ -17,11 +17,11 @@ Versioning follows the Semantic Versioning guidelines, with major, minor and pat
 Download using git:
 
 ```
-git clone -b release/0.19 --recurse-submodules https://github.com/LeelaChessZero/lc0.git
+git clone -b release/0.21 --recurse-submodules https://github.com/LeelaChessZero/lc0.git
```

Comment:
Maybe add a banner with large bold red flashing all-caps font:
```
REPLACE 0.21 WITH THE LATEST RELEASE VERSION
```
?

Reply:
Adding this for future reference (since the PR is already merged): This will only address the clone command, the archive download links are not showing the version number - If I don't find something a set that always gets the latest release, the phrasing should be revised.